pkg/telemetry/telemetry.go

```diff
@@ -142,7 +147,7 @@ func (t *engineImpl) Stop() {
 func (t *engineImpl) Start() {
 	t.stop = make(chan struct{})
 	go func() {
-		collect := time.NewTicker(time.Minute * 55) // Collect telemetry info every 55 minutes since housekeeper fills the _timescaledb_catalog.metadata table every 1 hour.
+		collect := time.NewTicker(time.Hour) // Collect telemetry info every hour.
```

Comment:
Might be worth adding some jitter to the ticker so that you don't have a bunch of instances that start together racing to perform the same work.

Reply:
+1 this

Reply:
Go tickers are not accurate with respect to time. They shift by few milliseconds in some cases, so I think that should be fine for us.